index.bs

```diff
@@ -1527,10 +1579,64 @@ agent-info-response = {
   1: agent-info ; agent-info
 }
 
+streaming-capabilities-request = {
+  request
+}
+
+streaming-capabilities-response = {
+  response
+  1: streaming-capabilities ; streaming-capabilities
+}
+
+streaming-capabilities-update = {
+  1: streaming-capabilities ; streaming-capabilities
+}
+
 agent-info = {
-  0: text ; friendly-name
+  0: text ; display-name
   1: text ; model-name
-  ; ...
+  ? 2: bool ; audio-only
+}
+
+resolution = {
+  0: uint ; horizontal-resolution
+  1: uint ; vertical-resolution
+  2: [1* float64] ; refresh-rates
+}
+
+codec = (
+  H-264: 0
+  MPEG-4: 1
+  MP3: 2
+  VP8: 3
+  AAC-LC: 4
+  AC3: 5
+  eAC3: 6
+  FLAC: 7
+  PCM: 8
+  Vorbis: 9
+)
+
+bit-rate-limits = {
+ 0: uint ; min-bit-rate
+ 1: uint ; max-bit-rate
+ 2: uint ; max-latency
```

Comment:
I don't think these should be bundled together.  It's much more common to want a max than a min bitrate.